Jenkinsfile_nightly

```diff
@@ -67,94 +67,114 @@ def teardownSecretsForIntegrationTests() {
-        yarnBuilder.yarn('install-dependencies')
-        env.MOCKED_PAYMENTS = false
-        env.TEST_RETRIES = 5
-        env.TESTS_FOR_ACCESSIBILITY = true
-        env.ENV = 'aat'
+    stage('Functional UI tests webkit ') {
+      try {
+        yarnBuilder.yarn('test:functional-webkit')
+      } catch (Error) {
+        unstable(message: "${STAGE_NAME} is unstable: " + Error.toString())
+      } finally {
+        publishHTML([
+          allowMissing         : true,
+          alwaysLinkToLastBuild: true,
+          keepAll              : true,
+          reportDir            : 'playwright-report',
+          reportFiles          : 'index.html',
+          reportName           : 'UITestWebkitReport'
+        ])
+        steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/functionalTest/**/*'
+      }
     }
 
-    afterAlways('fullFunctionalTest') {
-      stage('Functional UI tests chromium') {
-            try {
-                yarnBuilder.yarn('test:functional-chromium')
-            } catch (Error) {
-                unstable(message: "${STAGE_NAME} is unstable: " + Error.toString())
-            } finally {
-                publishHTML([
-                    allowMissing: true,
-                    alwaysLinkToLastBuild: true,
-                    keepAll: true,
-                    reportDir: 'playwright-report',
-                    reportFiles: 'index.html',
-                    reportName: 'UITestChromiumReport'
-                ])
-                steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/functionalTest/**/*'
-            }
-        }
-      stage('Functional UI tests firefox') {
-        try {
-          yarnBuilder.yarn('test:functional-firefox')
-        } catch (Error) {
-          unstable(message: "${STAGE_NAME} is unstable: " + Error.toString())
-        } finally {
-          publishHTML([
-            allowMissing: true,
-            alwaysLinkToLastBuild: true,
-            keepAll: true,
-            reportDir: 'playwright-report',
-            reportFiles: 'index.html',
-            reportName: 'UITestFirefoxReport'
-          ])
-          steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/functionalTest/**/*'
-        }
-      }
-      stage('Functional UI tests webkit ') {
-        try {
-          yarnBuilder.yarn('test:functional-webkit')
-        } catch (Error) {
-          unstable(message: "${STAGE_NAME} is unstable: " + Error.toString())
-        } finally {
-          publishHTML([
-            allowMissing: true,
-            alwaysLinkToLastBuild: true,
-            keepAll: true,
-            reportDir: 'playwright-report',
-            reportFiles: 'index.html',
-            reportName: 'UITestWebkitReport'
-          ])
-          steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/functionalTest/**/*'
-        }
+    stage('cafCassAPITest') {
```

Comment:
do you want to add this stage in both main and nightly pipeline?

Reply:
The new endpoints does not trigger any state changes in FPL service. These end points  are used only by cafcass user not by FPL service so having them nightly will be sufficient. If added to master build it going to add extra constrain on each release and doesn't add any values to service.

Reply:
ok, the stage is in Jenkins_cnp file as well, may be it can be removed from there then?

Reply:
Yes, I have removed Jenkin_cnp , so the tests run only in nightly builds.